main.py

```diff
@@ -1,17 +1,27 @@
-# main.py
+from flask import Flask, request, jsonify, make_response,render_template, redirect
+import yaml
 
-import webapp2
-from content import CONTENT
+DEBUG = True
+app = Flask(__name__)
+app.debug = DEBUG
 
-class Home(webapp2.RequestHandler):
-    def get(self):
-        
-        # Create links to the Login handler.
-        self.response.write(CONTENT)
+@app.route('/')
+def handler():
+    # load a YAML file with text content
+    with open("content.yaml", 'r') as stream:
+        try:
+            yaml_text = yaml.load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+    # parse the YAML file into a variables
+    idea1 = yaml_text['Idea1']
```

Comment:
this should definitely be down with a for-loop. Use `dict.items()` to loop through dictionary keys and values.